### PR TITLE
#68 rename to equibit on config files: configure, makefile, gitignore

### PR DESCRIPTION
**Pull request by r-soltani**
Renaming the binaries from Bitcoin to Equibit

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -14,10 +14,10 @@ AC_CONFIG_HEADERS([src/config/bitcoin-config.h])
...
 AC_CONFIG_MACRO_DIR([build-aux/m4])
 
-BITCOIN_DAEMON_NAME=bitcoind
-BITCOIN_GUI_NAME=bitcoin-qt
-BITCOIN_CLI_NAME=bitcoin-cli
-BITCOIN_TX_NAME=bitcoin-tx
+BITCOIN_DAEMON_NAME=equibitd
+BITCOIN_GUI_NAME=equibit-qt
+BITCOIN_CLI_NAME=equibit-cli
+BITCOIN_TX_NAME=equibit-tx
```
Comment (macterra) [inline configure.ac:20]: Why not change the names of the env variables too? (just curious)

Comment (r-soltani) [reply configure.ac:20]: that is my next step

**Review by macterra — CHANGES_REQUESTED** [review]
No checks yet. Maybe push a commit to trigger the continuous build?

**Comment by r-soltani** [comment]
The check will fail because the functional tests(i.e. python codes) are referencing the bitcoin binaries instead of equibit binary. This issue will be resolved by the next commits.

**Review by macterra — APPROVED** [review]

**Comment by macterra** [comment]
OK to make a temporary exception...

**Pull request merged**